service/backup/backupRunner.go

```diff
@@ -189,3 +196,56 @@ func (b *backupRunner) updateBackupStatus() {
 		b.Logger.Errorf("Coud not update backup resource: %v", updateErr)
 	}
 }
+
+// startPodTemplates will start and wait all pods defined in the template and
+// wait for at least one replication to be available for each pod.
+func (b *backupRunner) startPodTemplates() {
+	b.runningSets = b.getReplicaSets()
+	for _, set := range b.runningSets {
+		b.Logger.Infof("Creating command pod %v/%v\n", b.backup.GetNamespace(), set.GetName())
+		runningSet, err := b.K8sCli.Extensions().ReplicaSets(b.backup.GetNamespace()).Create(&set)
+		if err != nil {
+			b.Logger.Errorf("error creating command pod %v: %v\n", set.GetName(), err)
```

Comment:
I'm not really happy with the whole error handling. Can we at least expose failures as Prom metrics? Otherwise they will just silently fail (as no one looks at the logs).